docs/accordproject-concepts.md

```diff
@@ -5,31 +5,31 @@ title: Key Concepts
 
 ## What is an Accord Project Template?
 
-An Accord Project template ties legal text (for a clause or contract) to computer code. It is composed of three elements: 
+An Accord Project template ties legal text (for a clause or contract) to computer code. It is composed of three elements:
 
 - **The Template Text**: the natural language of the template
-- **The Template Model**: the data model that backs the template 
+- **The Template Model**: the data model that backs the template
 - **The Template Logic**: the executable business logic for the template
 
 ![Accord Project Template](assets/020/template.png)
 
-When combined these three elements allow templates to be edited, validated, verified, and then executed on any computer platform (on your own machine, on a Cloud platform, on Blockchain, etc).
+The three components (Text - Model - Logic) can also be intuitively understood as a **linear progression**, from _human-readable_ legal text to _machine-readable_ code. When combined these three elements allow templates to be edited, validated, verified, and then executed on any computer platform (on your own machine, on a Cloud platform, on Blockchain, etc).
```

Comment:
The mathematician in me is shaking its head. How about `progression` or `incremental progression`?

Reply:
Hahaha sounds good ;) (makes note to self - nothing with _linear_ next time...)

Reply:
> Hahaha sounds good ;) (makes note to self - nothing with _linear_ next time...)

Well I could suggest to use linear when it's actually _linear_ ! 😉